docs/src/administration/web/configure-project.md

```diff
@@ -11,23 +11,23 @@ description: |
 
 From the first page of the project settings, `General`, you can update the project name, or navigate to other project settings options on the left side of the screen.
 
-![configure project](/images/management-console/settings-basics.png)
+![configure project](/images/management-console/settings-basics.png "0.7")
 
 ## Access
 
 The `Access` screen allows you to manage users' access on your project.
 
 You can invite new users to your project by clicking the `Add` button and entering their email address, or modify permissions of existing users by clicking the `Edit` link when hovering over the user.
 
-![Project configure icon](/images/management-console/settings-project-access.png)
+![Project configure icon](/images/management-console/settings-project-access.png "0.7")
 
 {{< note >}}
 Currently, permissions changes that grant or revoke SSH access to an environment take effect only after the next time that environment is deployed.
 {{< /note >}}
 
 Selecting a user will allow you to either edit that user's permissions or delete the user's access to the project entirely.
 
-![Manage users of your Platform.sh project](/images/management-console/edit-user.png)
+![Manage users of your Platform.sh project](/images/management-console/edit-user.png "0.7")
```

Comment:
Is the fraction an image resize?

Reply:
yessir, scales it down

Reply:
Aw, saw that in the template further down.  Nifty.